web/sites/default/settings.php

```diff
@@ -51,7 +51,7 @@
 $config['geocoder.settings']['plugins_options']['googlemaps']['apikey'] = getenv('GOOGLE_MAP_API_SERVER_KEY');
 
 // Environment indicator defaults.
-$env_colour = !empty(getenv('SIMPLEI_ENV_COLOR')) ? getenv('SIMPLEI_ENV_COLOR') : '#000000';;
+$env_colour = !empty(getenv('SIMPLEI_ENV_COLOUR')) ? getenv('SIMPLEI_ENV_COLOUR') : '#000000';
```

Comment:
🤣  💯

Reply:
Annoyingly, this isn't consistent across the projects. I suspect I'm to blame as well... in hindsight, I'd use en-GB spelling wherever possible.